arch/arm/boot/dts/overlays/g2labs-dac-overlay.dts

```diff
@@ -0,0 +1,34 @@
+// Definitions for G2LABS DAC
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "brcm,bcm2708";
+
+	fragment@0 {
+		target = <&i2s>;
+		__overlay__ {
+			status = "okay";
+		};
+	};
+
+	fragment@1 {
+		target-path = "/";
+		__overlay__ {
+			tda1543-codec {
+				#sound-dai-cells = <0>;
+				compatible = "ti,tda1543";
```

Comment:
compatible="<manufacturer>,<model>" Texas Instruments manufacture/manufactured the TDA1543? You learn something new every day! ;)